src/main/java/com/newts/newtapp/entities/User.java

```diff
@@ -88,6 +88,13 @@
     @Type(type = "list-array")
     private ArrayList<Integer> conversations;
 
+    /**
+     * A list of user identifiers corresponding to the Users that have been blocked by this user.
+     */
+    @Column(name = "blockedUsers", columnDefinition = "int[]")
```

Comment:
column naming convention says should be `blocked_users` in the database

Reply:
Also, add to the README.md such that the database is initialized with this new column.